action.yml

```diff
@@ -1,6 +1,6 @@
-name: "Contains tag"
+name: "Contains tag Fork"
 description: "Check if branch of commit contains tag."
-author: "Rick Staa"
+author: "Rick Staa Forked by Sebastian Neumann"
```

Comment:
This is probably unneeded in this repo

Reply:
I have Released a Fork into the Marketplace to use the Action today